Bottom: 847bd975bd5e8663708bd9df45ee6b7bfb0ea603
Top:    f9167b816814b1b5b28a778cd068df615d1dfd3a
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-06-04 04:43:30 -0300

Refresh of remaining-unary-exprs.patch

---

diff --git a/include/gcc-cp-interface.h b/include/gcc-cp-interface.h
index 69d3ec9..0ebbfe7 100644
--- a/include/gcc-cp-interface.h
+++ b/include/gcc-cp-interface.h
@@ -41,7 +41,7 @@ struct gcc_cp_context;
 
 enum gcc_cp_api_version
 {
-  GCC_CP_FE_VERSION_0 = 0xffffffff-14
+  GCC_CP_FE_VERSION_0 = 0xffffffff-15
 };
 
 /* Qualifiers.  */
